drizzlepac/haputils/processing_utils.py

```diff
@@ -50,6 +50,13 @@ def get_rules_file(product, rules_type=""):
         exposure.  Valid values: blank/empty string (''), 'SVM' or 'svm'
         for SVM processing (default) and 'MVM' or 'mvm' for MVM processing.
 
+    rule_root : str, optional
```

Comment:
Just a typo - rule_root should be rules_root.

Reply:
Fixed in a625671.